pages/base_page.py

```diff
@@ -0,0 +1,39 @@
+from selenium.webdriver.support.wait import WebDriverWait  # type: ignore
+from selenium.webdriver.support import expected_conditions as EC  # type: ignore
+
+
+class BasePage:
+
+    def __init__(self, driver, link):
+        self.driver = driver
+        self.link = link
+
+    def get_text(self, locator):
+        return self.find_element(locator).text
+
+    def get_texts(self, locator):
+        return self.find_elements(locator)
+
+    def click(self, locator):
+        element = self.find_element(locator)
+        element.click()
```

Comment:
Можно сократить код, либо отказаться от данного метода совсем, так как он не совершает особо сложных действий над элементом.
```suggestion
    def click(self, locator):
        self.find_element(locator).click()
```